pkg/distributor/ha_tracker_test.go

```diff
@@ -792,6 +791,58 @@ func TestCheckReplicaCleanup(t *testing.T) {
 	))
 }
 
+// TestStartingSync tests that values are synced to KVStore on startup
+func TestStartingSync(t *testing.T) {
+	t.Run("check that if a second replica is set up during preLoad that it does not write to it since its not the leader", func(t *testing.T) {
```

Comment:
this test description looks out of place. Since there's only one test case here can we remove the t.Run?

Reply:
yeah i can.